solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -436,7 +439,10 @@ protected void runCloudExample(CommandLine cli) throws Exception {
 
     boolean prompt = !cli.hasOption("noprompt");
     int numNodes = 2;
-    int[] cloudPorts = new int[] {8983, 7574, 8984, 7575};
+    int defaultPort =
+        Integer.parseInt(
+            cli.getOptionValue('p', SystemUtils.getEnvironmentVariable("SOLR_PORT", "8983")));
```

Comment:
do we need systemutils? 

```
System.getenv().getOrDefault("SOLR_PORT", "8983");
```

Reply:
Ah, I just tried `System.getEnv()`, will switch.